SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchNoteWindow.tsx

```diff
@@ -189,8 +189,13 @@ const NoteWidget: React.FC<SEBrowser.IWidget<ISetting>> = (props) => {
                     />
                     <Select<OpenXDA.Types.NoteType>
                         Record={noteType}
-                        Label={'Type'}
-                        Options={noteTypes.map(t => ({ Label: t.Name, Value: t.ID.toString() }))}
+                        Label={'Record:'}
+                        Options={[
+                                { Label: 'Event', Value: '1' },
+                                { Label: 'Meter', Value: '2' },
+                                { Label: 'Asset', Value: '3' },
+                                { Label: 'Substation', Value: '4' }
+                            ]}
```

Comment:
`Event` and `Meter` seem to be flip-flopped here. It's probably best not to hard-code the `Value` in case there are discrepancies in the `ID` field in other databases.

Reply:
Erika is right that hardcoding causes issues. Note that the `RootFilter` you added in `NoteTypeControler` will already take care of filtering these so no need for this change at all. We are also adding a `Label` property that can be used in the database once that is complete (may require a sepperate PR in the future)